main.py

```diff
@@ -0,0 +1,79 @@
+# bot.py
+import os
+import random
+import RockPaperScissors as rps
+import discord
+from dotenv import load_dotenv
+
+load_dotenv()
+TOKEN = os.getenv('DISCORD_TOKEN')
+
+client = discord.Client()
+
+@client.event
+async def on_ready():
+    print(f'{client.user.name} has connected to Discord!')
+
+@client.event
+async def on_member_join(member):
+    await member.create_dm()
+    await member.dm_channel.send(
+        f'Hi {member.name}, welcome to my Discord server!'
+    )
+
+# A Default event
+@client.event
+async def on_message(message):
+    if message.author == client.user:
+        return
+
+    if message.content == '99!':
+        await message.channel.send('test')
+
+# Play the classic game of Rock, Paper, Scissors
```

Comment:
can probably make this smaller in the future. try to move logic into RockPaperScissors.py for a cleaner main.py